python_modules/dagster/dagster/_api/snapshot_sensor.py

```diff
@@ -21,6 +21,7 @@ def sync_get_external_sensor_execution_data_ephemeral_grpc(
     last_completion_time: Optional[float],
     last_run_key: Optional[str],
     cursor: Optional[str],
+    last_start_time: Optional[float] = None,
```

Comment:
name bikeshedding:

just confirming this is the last time the *sensor* was started right? (as opposed to the last time a tick was started)
maybe worth clarifying that in the name

would "sensor_start_time" be better? Maybe "last" is implied in that case? Or maybe "last_sensor_start_time" or "most_recent_sensor_start_time" is maximally clear